contrib/offline/manage-offline-container-images.sh

```diff
@@ -98,6 +87,11 @@ function register_container_images() {
 		exit 1
 	fi
 
+	# To avoid "no such host" error.
+	if ! grep -q ${LOCALHOST_NAME} "/etc/hosts"; then
+		echo "127.0.0.1 ${LOCALHOST_NAME}" >> /etc/hosts
```

Comment:
I'm not a big fan of messing with `/etc/hosts` in this way.

Reply:
In general, a local host name should exist in /etc/hosts.
I am not sure why `no such host` error can happen.